File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-15/source/2026-07-15-reliability-runbook-log-gap-automation.py
"""Northstar Transit Cloud; Checkout SVC-17; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Checkout SVC-17"
INCIDENT = "INC-260713"
TOPIC = "log_gap"
SPINE_IDS = ['org.ntc', 'svc.identity.svc23', 'rel.inc260713.timeline', 'date.2026-07-15', 'term.no-kio-claim']
FINGERPRINT = "e27483fd231c463f"

def reconcile_export_gap(minutes: int, reconciled_by: str) -> dict[str, object]:
    return {'gap_minutes': minutes, 'reconciled_by': reconciled_by, 'status': 'reconciled'}

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-15/source/2026-07-15-reliability-runbook-log-gap-automation.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "log_gap";
pub const FINGERPRINT: &str = "3db4d390c08c4371";
pub fn export_gap_reconciled(minutes: u8) -> bool { minutes == 7 }
